examples/cloudml-sklearn-pipeline/scripts/deploy.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+#
+# Convenience script for deploying trained scikit-learn model.
+#
+# Prerequisites:
+#   - Google Cloud SDK
+#
+# Globals:
+#   PROJECT_ID: Google Cloud project to use.
+#
+# Arguments:
+#   $1: Path to directory containing trained and exported scikit-learn model
+#   $2: Name of the model to be deployed
+#   $3: Version of the model to be deployed
+
+MODEL_NAME=$1
+VERSION_NAME=$2
+JOB_DIR=$3
+
+REGION=us-central1
+FRAMEWORK=SCIKIT_LEARN
+RUN_TIME=1.13
+PYTHON_VERSION=3.5 # only support python 2.7 and 3.5
```

Comment:
A few times throughout the doc Python 3.5 is mentioned, I would make clear throughout which version is used instead of mentioning options

Reply:
Both version are supported actually. I have added explanation in the readme to make it clearer as "Note that both Python 2.7 and Python 3.5 are supported, but Python 3.5 is the recommended one since 2.7 is [deprecated](https://pythonclock.org/) soon"

Reply:
Can you check this is the same across all files, e.g. in the config Yaml I still see a different convention. What I meant was across all files (including Readme) make clear what version you run.

Reply:
Thanks for the comment. I have checked all, and ensure the message is consistent.